tayoto_mikaellaantonette/script.js

```diff
@@ -1,17 +1,63 @@
+const form = document.querySelector("form");
+const nameInput = document.querySelector("name");
+const commentInput = document.querySelector("gcomment");
+const commentList = document.querySelector("comment-list");
+const sortAscButton = document.querySelector("sort-asc");
+const sortDescButton = document.querySelector("sort-desc");
```

Comment:
Your comment section is not working; it seems like you are using the wrong syntax here. If you are referring to an ID, it should be a hash character followed by your ID name.

`const nameInput = document.querySelector("#name");`

Reference: https://www.w3schools.com/html/html_id.asp#:~:text=Using%20The%20id%20Attribute&text=It%20is%20also%20used%20by,properties%20within%20curly%20braces%20%7B%7D.